platform/solr/pom.xml

```diff
@@ -23,9 +23,6 @@
     <artifactId>solr</artifactId>
     <packaging>pom</packaging>
     <name>DDF Solr</name>
-    <properties>
-        <saxon.version>9.5.1-5</saxon.version>
```

Comment:
I just confirmed that this property isn't used anywhere, so I removed it after checking with @pklinef.